### PR TITLE
make backdrop div 100% width/height

### PR DESCRIPTION
**Pull request by madduccino**
#338 - reproduced the bug in /create by adding doctype. Added 100% width/height to html and body tags and #project and #backdrop divs which resolved it.

### CODE REVIEW

**Comment by stevekrouse** [comment]
View this pull request at:

https://cdn.rawgit.com/stevekrouse/WoofJS/a7e491626756353c5830b2576876070f63174ac9/index.html

File: src/woof.es6.js [hunk 1]
```diff
@@ -95,11 +95,17 @@ function Woof({global = false, fullScreen = false, height = 500, width = 350} =
   
   window.addEventListener("load", () => {
     // create the main div that Woof lives in
```
Comment (stevekrouse) [inline src/woof.es6.js]: This comment is now out of place. Can you put it where it belongs, above `thisContext._mainDiv = document.createElement("div");`?

File: src/woof.es6.js [hunk 2]
```diff
@@ -95,11 +95,17 @@ function Woof({global = false, fullScreen = false, height = 500, width = 350} =
...
+    document.documentElement.style.width = "100%";
+    document.documentElement.style.height = "100%";
```
Comment (stevekrouse) [inline src/woof.es6.js]: I've never seen `document.documentElement` before. Do you need it here? It seems like the `document.body` code below should take care of this, no?

File: create.html [hunk 3]
```diff
@@ -613,6 +613,13 @@
         
         iframe.src = ""; // first, we clear the preview iframe
         setTimeout(function() {
+        
+            // add doctype to iframe
+            var myContent = '<!DOCTYPE html>'
+            iframe.contentWindow.document.open('text/htmlreplace');
```
Comment (stevekrouse) [inline create.html]: I can't find anywhere online to support `text/htmlreplace`. It seems like you can leave it out as it says here https://www.w3schools.com/jsref/met_doc_open.asp

File: src/woof.es6.js (hunk 2 above)
Comment (madduccino) [inline src/woof.es6.js]: document.documentElement adds style to the html tag which is necessary - 100% width/height has to be set on html and body.

**Pull request merged**